versioned_docs/version-0.x.0/kubeslice-cli/kubeslice-cli-overview.mdx

```diff
@@ -0,0 +1,20 @@
+# KubeSlice CLI Overview
+
+## Introduction 
+Kubeslice-cli is a command-line tool that allows you to perform Kubeslice operations on Kubernetes clusters. It simplifies the work need to 
```

Comment:
Kubeslice-cli should be kubeslice-cli

Reply:
this comment is not fixed - `the work needed to` or `the work required to`

Reply:
sorry, overread it. Fixed it.